### PR TITLE
Adding error code 5012

### PR DESCRIPTION
**Pull request by AlexJerabek**

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/javascript-api-for-office-error-codes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-ErrorCodes/docs/reference/javascript-api-for-office-error-codes.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C27%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201811271859195770-581%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Pklg%2BTn1hefROGUojCoorvmzRAV%2Bk6ztgD%2BmPTRmvCU%3D&st=2018-11-27T18%3A56%3A09Z&se=2018-12-28T19%3A01%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/javascript-api-for-office-error-codes.md](https://github.com/OfficeDev/office-js-docs-pr/blob/AlexJ-ErrorCodes/docs/reference/javascript-api-for-office-error-codes.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C27%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201811271859472575-581%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ZGuGoHuOyiftlyXl6gEQpLibRwZShiEv0VSZcufG8OU%3D&st=2018-11-27T18%3A57%3A40Z&se=2018-12-28T19%3A02%3A40Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/javascript-api-for-office-error-codes.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ The following table lists the error codes, names, and messages displayed, and th
 |5006|Session Timeout|The document session timed out. Reload the document. |The session has timed out.|
 |5007|Invalid API call|The enumeration is not supported in the current context.|The enumeration is not supported in the current context.|
 |5009|Permission Denied|Access Denied|The add-in does not have permission to call the specific API.|
+|5012|Invalid Or Timed Out Session|Your Office Online session has expired or is invalid. To continue, refresh the page.|The session between the office client and server has expired or when the date, time, or time zone is incorrect on your computer.|
```
Comment (kbrandl) [inline docs/reference/javascript-api-for-office-error-codes.md]: Capitalize: "the office client" -> "the Office client"

Comment (kbrandl) [reply docs/reference/javascript-api-for-office-error-codes.md]: Reword the second part of the last sentence to remove the word "when" -> "The session between the office client and server has expired or the date, time, or time zone is incorrect on your computer."

**Pull request merged**